app/controllers/users_controller.rb

```diff
@@ -100,16 +100,16 @@ def update
       params[:user][:name] = @user.name
       params[:user][:email] = @user.email
     end
-
-    if @user.update_attributes(user_params)
-      @user.update_attributes(email_verified: false) if user_params[:email] != @user.email
-
-      # Mark password as secure
-      @user.update(secure_password: true)
```

Comment:
Removed to fix #3078

Reply:
Is this still needed with https://github.com/bigbluebutton/greenlight/pull/3101 merged?